tests/media/server/ipc/sessionManagementServer/SessionManagementServerTestsFixture.h

```diff
@@ -55,11 +56,16 @@ class SessionManagementServerTests : public testing::Test
     void sendServerInitialize();
     void sendServerInitializeAndExpectFailure();
     void sendServerStart();
+    void testSocketOwnershipValidOwnerAndGroup();
```

Comment:
Can these be renamed to follow the convention in this file, e.g. first one something like:
serverWillInitializeWithValidOwnerAndGroup

Can then move these functions next to the previously definded serverWill/clientWill funcitons.